libs/challenge-registry/ui/src/lib/footer/_footer-theme.scss

```diff
@@ -23,13 +22,30 @@
 
 @mixin typography($theme) {
   $typography-config: mat.get-typography-config($theme);
+  $font-family: map.get($typography-config, 'font-family');
```

Comment:
Not sure if it's only on my end, but the `Poppins` font doesn't seem to reflect on the app:
<img width="1164" alt="Screen Shot 2022-06-24 at 12 29 15 AM" src="https://user-images.githubusercontent.com/9377970/175486005-eececb34-7bde-4ea8-a41f-c3896dd555e0.png">

Reply:
P.S. I do see it being used in the navbar though! 🤣

Reply:
Good catch! I thought it worked when I see font-family was using `Poppins` on footer, but it should not contain quotes (either single or double quotes).
<img width="1380" alt="Screen Shot 2022-06-24 at 9 56 31 AM" src="https://user-images.githubusercontent.com/73901500/175550945-53e763ad-2bb3-484a-86de-5fd3dec1c8cf.png">

> I do see it being used in the navbar though! 
I don't know why navbar is smarter than footer 🤣

Reply:
Both of below ways are working:
1. 
```scss
@mixin typography($theme) {
  $typography-config: mat.get-typography-config($theme);
  $font-family: map.get($typography-config, font-family);
  footer {
    font-family: $font-family;
  }
}
``` 
We would also need to replace`$font-family: "'Poppins', 'sans-serif'"` with `$font-family: 'Poppins, sans-serif'` in this way.

2. 
```scss
@mixin typography($theme) {
  $typography-config: mat.get-typography-config($theme);

  footer {
    font-family: mat.font-family($typography-config);
  }
```

I will update to use the second way since it is shorter.